MFRC522.cpp

```diff
@@ -820,6 +820,62 @@ MFRC522::StatusCode MFRC522::PICC_HaltA() {
 	return result;
 } // End PICC_HaltA()
 
+/**
+ * Transmits a Request command for Answer To Select (ATS).
+ *
+ * @return STATUS_OK on success, STATUS_??? otherwise.
+ */
+MFRC522::StatusCode MFRC522::PICC_RATS(byte *bufferATS,	///< The buffer to store the ATS (Answer to select) in
```

Comment:
Sorry... FSD=128 is a mistake... it should say FSD=64. It was hard coded that way to tell the card the maximum frame size our reader can handle, which is 64 if I'm not mistaken... The value however is correct 0x5X stands for FSD=64 bytes